File: scratch/list_clones.py
import os

clones_path = os.path.join(os.path.dirname(__file__), 'clones')
dirs = os.listdir(clones_path)

for d in dirs:
    dir_path = os.path.join(clones_path, d)
    if not os.path.isdir(dir_path):
        continue
        
    print(f"\n========================================")
    print(f"REPO: {d}")
    print(f"========================================")
    
    for root, subdirs, files in os.walk(dir_path):
        if '.git' in subdirs:
            subdirs.remove('.git')
        for f in files:
            full_path = os.path.join(root, f)
            rel_path = os.path.relpath(full_path, dir_path)
            size_kb = os.path.getsize(full_path) / 1024
            print(f"- {rel_path} ({size_kb:.2f} KB)")

File: scratch/list_clones.js
const fs = require('fs');
const path = require('path');

const clonesPath = path.join(__dirname, 'clones');
const dirs = fs.readdirSync(clonesPath);

dirs.forEach(dir => {
    const dirPath = path.join(clonesPath, dir);
    if (!fs.statSync(dirPath).isDirectory()) return;
    
    console.log(`\n========================================`);
    console.log(`REPO: ${dir}`);
    console.log(`========================================`);
    
    const files = [];
    function walk(currentPath) {
        const list = fs.readdirSync(currentPath);
        list.forEach(file => {
            const filePath = path.join(currentPath, file);
            const stat = fs.statSync(filePath);
            if (stat.isDirectory()) {
                if (file !== '.git') {
                    walk(filePath);
                }
            } else {
                files.push({
                    rel: path.relative(dirPath, filePath),
                    size: stat.size
                });
            }
        });
    }
    
    try {
        walk(dirPath);
        files.forEach(f => {
            console.log(`- ${f.rel} (${(f.size/1024).toFixed(2)} KB)`);
        });
    } catch (e) {
        console.log(`Error walking: ${e.message}`);
    }
});
